src/hibernate.cfg.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+	<session-factory>
+
+		<property name="connection.driver_class">org.postgresql.Driver</property>
```

Comment:
this connection properties will, in a future task, be moved to payara (jdbc) and then just referenced here. Not necessary by now

Reply:
Got it